plugins/src/webrtcsink/imp.rs

```diff
@@ -1134,6 +1160,31 @@ impl InputStream {
     }
 }
 
+impl NavigationEventHandler {
+    pub fn new(
+        element: &super::WebRTCSink,
+        webrtcbin: &gst::Element,
+    ) -> Self {
+
+        let channel = webrtcbin.emit_by_name::<WebRTCDataChannel>(
+            "create-data-channel",
+            &[&"input", &gst::Structure::new_empty("nothing")],
```

Comment:
```suggestion
            &[&"input", &None::<gst::Structure>],
```

Do you have to actually pass an empty dummy structure here?